### ISSUE TITLE
Crashing in Timesplitters 2 with Windows host and Linux client

### ISSUE THREAD

**Issue by kylemiller3** [TheLastRar/CLR-DEV9#13]
So I was able to join the lobby of a Windows host (but not the other way around). However, the connection drops and it throws an exception on both machines.

For Linux:
```
Object reference not set to an instance of an object
  at System.Net.Sockets.UdpClient.get_Available () [0x00000] in <e8eb3d7a311640f484845e45cbec8973>:0 
  at CLRDEV9.DEV9.SMAP.Winsock.Sessions.UDPServerSession.Recv () [0x00001] in <baacf1ca82a44f67b6e67661b444aac2>:0 
  at CLRDEV9.DEV9.SMAP.Winsock.Winsock.Recv (CLRDEV9.DEV9.SMAP.Data.NetPacket& pkt) [0x00082] in <baacf1ca82a44f67b6e67661b444aac2>:0 
  at CLRDEV9.DEV9.SMAP.AdapterManager.NetRxThread () [0x0002a] in <baacf1ca82a44f67b6e67661b444aac2>:0 
```

Windows:
![exception](https://user-images.githubusercontent.com/5983525/56434811-3c757500-62a4-11e9-8713-1adefe77e1d3.png)

**Comment by TheLastRar** [comment]
Can you test this build?
https://github.com/TheLastRar/CLR-DEV9/releases/tag/v0.8.6PR1

**Comment by TheLastRar** [comment]
Assumed fixed in v0.8.6

That build addressed a similar issue in another game

Please reopen if this is not the case.
